Makefile

```diff
@@ -1,28 +1,77 @@
-.PHONY: compile test unit-test integration-test integration-test-timeseries protogen
+.PHONY: all clean compile protogen configure-security-certs help
+.PHONY: test unit-test integration-test
+.PHONY: integration-test-timeseries integration-test-hll integration-test-security
 
-RIAK_PORT ?= 8087
+PROJDIR = $(realpath $(CURDIR))
+RESOURCES_DIR = $(PROJDIR)/src/test/resources/
+CA_DIR = $(PROJDIR)/tools/test-ca
+CERTS_DIR = $(CA_DIR)/certs
+PRIVATE_DIR = $(CA_DIR)/private
+RIAK_PORT ?= 808f
+RUN_YOKOZUNA ?= true
+
+all: test
+
+clean:
+	mvn clean
 
 compile:
-	mvn clean compile
+	mvn compile
 
 test: unit-test integration-test
 
 test-ts: unit-test integration-test-timeseries
 
-unit-test:
+unit-test: compile
```

Comment:
The maven `compile` phase is in the lifecycle of the `test` phase, so we don't need to explicitly call it.  Worse case we compile twice.